awxkit/awxkit/api/pages/unified_job_templates.py

```diff
@@ -19,6 +19,10 @@ class UnifiedJobTemplate(HasStatus, base.Base):
         'job_type',
         'verbosity',
         'inventory',
+        'forks',
+        'timeout',
+        'job_slice_count',
+        'execution_environment',
```

Comment:
What about instance groups?

Reply:
we have a method to add instance_groups and labels: https://github.com/ansible/awx/blob/feature-prompt-on-launch-on-templates/awxkit/awxkit/api/pages/schedules.py#L50-L56
Methods use POST to associate labels and ig, and those fields are passed with the Schedules payload.